default/editable-data-grid/README.md

```diff
@@ -0,0 +1,51 @@
+# Editable Data Grid Sample Extension
+
+This is a sample extension that demonstrates a workaround solution for making the `vscode-data-grid` component editable.
+
+![A screenshot of the sample extension.](./assets/editable-data-grid.png)
+
+## Context
+
+> **Warning**<br>
+> This is not a fully tested (nor first-party) editable data grid. Proceed with caution.
+
+There have been many requests (such as [this issue](https://github.com/microsoft/vscode-webview-ui-toolkit/issues/493)) asking for a more interactive data grid.
+
+The Webview UI Toolkit is implemented using a set of unstyled based components from [FAST](https://fast.design). They offer a `data-grid` component which, at this time, statically renders data in a table-like format but does not implement any APIs for interactivity or editability.
+
+Because FAST owns the underlying data grid implementation, first-party support for interactivity will need to come from them. With that said, workaround solutions are possible and this sample extension demonstrates one such solution (initially inspired by another solution from [@worksofliam](https://github.com/worksofliam)).
```

Comment:
@worksofliam I'm assuming it is, but wanted to double check to see if it's cool if I give you credit here for having come up with the original editable data grid solution?

Reply:
Well, I didn't come up with `contenteditable`, but I did combine it with the data grid you supplied! (first, I guess?)

Reply:
Haha yeah, I'm referring to the overall solution of using `contenteditable` to make the `vscode-data-grid` interactive

Reply:
But yeah, I just wanted to get your permission to link your GH profile in this README –– I've run into some people in the past who were less keen to have their name/profile listed in a more public/higher visibility location so I always like to check with folks first :)

Reply:
@hawkticehurst No problem! Thank you for asking. I appreciate it.

Reply:
Of course!